### PR TITLE
fix add insurance company from demographics

### LINKED ISSUE #5176: add insurance company from demographics
### Describe the bug
add an insurance company from demographics
Screenshot from 2022-04-17 10-16-30

the insurance doesn't populate since there's no id being returned [here](https://github.com/openemr/openemr/commit/cbb795e92db34f01acc791e699eecdb96d2d6557#diff-4f8cda1691e039bc4819d01e6f5ac2a9c45baedc41edd47e88b37dd17f003b25L243-L245)

[sql.inc](https://github.com/openemr/openemr/blob/master/library/sql.inc#L305-L323`) returns the correct last id generated

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #5176 and #5180

### CODE REVIEW

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
nice! mystery solved!

tenor (67)

**Comment by stephenwaite** [comment]
what do you think @sjpadgett?

File: src/Services/InsuranceCompanyService.php [hunk 1]
```diff
@@ -214,7 +219,9 @@ public function getInsuranceClaimTypes()
 
     public function insert($data)
     {
-        $freshId = $this->getFreshId("id", "insurance_companies");
+        // insurance companies need to use sequences table since they share the
+        // addresses table with pharmacies
+        $freshId = generate_id();
```
Comment (sjpadgett) [inline src/Services/InsuranceCompanyService.php:224]: Of the two ways to go, this is the best imo and obviously yours.

**Review by sjpadgett — APPROVED** [review]
Nice job Stephen!

File: src/Services/InsuranceCompanyService.php (hunk 1 above)
Comment (stephenwaite) [reply src/Services/InsuranceCompanyService.php:224]: thanks for reviewing and the advice this weekend!

**Pull request merged**